.github/workflows/release.yml

```diff
@@ -202,3 +211,5 @@ jobs:
           password: ${{ secrets.DOCKER_PASSWORD }}
           repository: supabase/realtime
           tags: latest,v${{ env.VERSION_AFTER }}
```

Comment:
Looks like it didn't push a new tag - perhaps we need 2 `Upload build to Docker Hub` steps, each with different architectures/tags?

Reply:
I think it's because it's using `docker/build-push-action@v1` instead of `v2`. Will address it in another PR.

Reply:
@kiwicopple looks like it's fine to define multiple platforms in a single step (see: https://github.com/docker/build-push-action/blob/master/docs/advanced/multi-platform.md).

@samtgarson i think we'll have to use `v2` of  `docker/build-push-action` to get this to work.

Reply:
update: looks like @soedirgo beat me to it! See https://github.com/supabase/realtime/commit/eed8a4dbec310069f3eaad68319d70f0b8e3dcd6#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R217